lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+'''
+Unit Tests for the manage.py command rename_user
+'''
+from django.core.management.base import CommandError
+import unittest
+from django.contrib.auth.models import User
+from pymongo import MongoClient
+from student.tests.factories import UserFactory
+import datetime
+from .. import rename_user
+import sys
+from StringIO import StringIO
+
+
+class TestUsernameRenameCommand(unittest.TestCase):
+    '''
```

Comment:
Use double-quotes for docstrings, per the style guide [1].

[1] https://github.com/edx/edx-platform/wiki/Python-Guidelines